.github/workflows/pr-checker.yml

```
@@ -0,0 +1,33 @@
+
+jobs:
+  check_pr_description:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Check PR Description
+        id: check_pr_description
+        run: |
```

actionlint

"github.event.pull_request.body" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions for more details

(expression)